gsoc/gsoc_sciml.md

```diff
@@ -87,48 +87,19 @@ is helpful but not required.
 
 **Mentors**: [Ludovico Bessi](https:https://github.com/ludoro), [Chris Rackauckas](https://github.com/ChrisRackauckas)
 
-## Parameter estimation for nonlinear dynamical models
-
-Machine learning has become a popular tool for understanding data, but scientists
-typically understand the world through the lens of physical laws and their
-resulting dynamical models. These models are generally differential equations
-given by physical first principles, where the constants in the equations such
-as chemical reaction rates and planetary masses determine the overall dynamics.
-The inverse problem to simulation, known as parameter estimation, is the process
-of utilizing data to determine these model parameters.
-
-The purpose of this project is to utilize the growing array of statistical,
-optimization, and machine learning tools in the Julia ecosystem to build
-library functions that make it easy for scientists to perform this parameter
-estimation with the most high-powered and robust methodologies. Possible projects
-include improving methods for Bayesian estimation of parameters via Stan.jl
-and Julia-based libraries like Turing.jl, or global optimization-based approaches.
-Novel techniques like classifying model outcomes via support vector machines
-and deep neural networks can also be considered. Research and benchmarking
-to attempt to find the most robust methods will take place in this project.
-Additionally, the implementation of methods for estimating structure, such
-as [topological sensitivity analysis](https://www.pnas.org/content/111/52/18507)
-along with performance enhancements to existing methods will be considered.
-
-Some work in this area can be found in
-[DiffEqParamEstim.jl](https://github.com/SciML/DiffEqParamEstim.jl)
-and [DiffEqBayes.jl](https://github.com/SciML/DiffEqBayes.jl). Examples
-can be found [in the DifferentialEquations.jl documentation]( https://docs.sciml.ai/dev/analysis/parameter_estimation).
+## Improved interfacing between ModelingToolkit.jl and GalacticOptim.jl
```

Comment:
Maybe move to the new numerics page?

Reply:
Doesn't seem like a good fit there? Maybe the Symbolics section would be better?

Reply:
Yeah that seems good.